### DISCUSSION TITLE
mmi plus no patch

### DISCUSSION THREAD

**Discussion by luigi1980** [Mr-MIBonk/M.I.B._More-Incredible-Bash#50] [Q&A] [answered]
Hi I already wrote in generic but I have not had an answer I explain here if you allow me ..... I tried to patch my MMI Plus with firmawer updated to MHI2Q_ER_AUG22_P5092_MU1329 but I can not I tried all the mib available but the error I see in the log always every time I try again is "MMX failed" and does not install anything , I attach the log ...... I would be grateful if you could help me 
[MU1329-JTB-07424.01.17118308A2.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/9903623/MU1329-JTB-07424.01.17118308A2.log)

**Comment by luigi1980** [comment]
[MU1329-JTB-07424.01.17118308A2 (1).log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/9920447/MU1329-JTB-07424.01.17118308A2.1.log)
il log di prima non si legge questo dovrebbe andare

**Comment by luigi1980** [comment]
I noticed that after installed the mib cmq the USB socket does not work both in full and in data only charging thanks again

**Comment by harman-f** [comment]
Both log files have 0 bytes after download.
Please upload them again.

**Comment by luigi1980** [comment, marked as answer]
__  __     ___           ___  
|  \/  |   |_ _|         | _ ) 
| |\/| |    | |          | _ \ 
|_|  |_|ore|___|ncredible|___/ash - V3.3.0 "Bentley Edition - BYG24"
______________________________________________________________________

NOT FOR COMMERCIAL USE - IF YOU BOUGHT THIS YOU GOT RIPPED OFF

Logging_MU1329-MIB2Q_ER_AUG22_P5092-JTB-07424.01.17118308A2:
HW Number: 033 - PN: 8W5035036
Variant: FMQ-P-TNL-EU-AU-MLE
EEPROM Variant: MU-P-LN-EU
Dataset Version: 8W0909545L
Model ID: 019402 - Car Model: Audi A4 (8W0)
VIN -> CAR: WAUZZZF41HA120002 - MIB: WAUZZZF41HA120002

 ls --> total 2532
      32 drwxrwxrwx 10 root      root          16384 Oct 16 11:23 .
      32 drwxrwxrwx 10 root      root          16384 Oct 16 11:23 ..
      32 drwxrwxrwx  2 root      root          16384 Oct 16 11:23 .System Volume Information
       2 -rwxrwxrwx  1 root      root            986 Jun 16 02:34 .git-blame-ignore-revs
       1 -rwxrwxrwx  1 root      root            278 Jun 16 02:34 .gitattributes
       1 -rwxrwxrwx  1 root      root            126 Jun 16 02:34 .gitignore
    1703 -rwxrwxrwx  1 root      root         871810 Jun 16 02:34 2022_06_MHI2_Password_List_V4.0.pdf
       0 n---------  1 root      root              0 Oct 16 11:23 =MIB
      81 -rwxrwxrwx  1 root      root          41255 Jun 16 02:34 GEM.png
      30 -rwxrwxrwx  1 root      root          14939 Jun 16 02:34 LICENSE
      32 drwxrwxrwx  4 root      root          16384 Jun 16 02:34 Launcher
       1 -rwxrwxrwx  1 root      root             60 Jun 16 02:34 M.I.B. - More Incredible Bash - Mibsolution Wiki.url
       0 -rwxrwxrwx  1 root      root              0 Jun 16 02:34 NOT FOR COMMERCIAL USE - IF YOU BOUGHT THIS YOU GOT RIPPED OFF.txt
     284 -rwxrwxrwx  1 root      root         144962 Jun 16 02:34 PATCH COMPATIBILITY TABLE.pdf
      10 -rwxrwxrwx  1 root      root           4899 Jun 16 02:34 README.md
       1 -rwxrwxrwx  1 root      root             33 Jun 16 02:34 VERSION
       1 -rwxrwxrwx  1 root      root            296 Jan 01  1980 WMPInfo.xml
      32 drwxrwxrwx  3 root      root          16384 Jun 16 02:34 apps
      32 drwxrwxrwx  4 root      root          16384 Jan 01  1980 backup
      77 -rwxrwxrwx  1 root      root          39351 Jun 16 02:34 bash.png
      32 drwxrwxrwx  2 root      root          16384 Jun 16 02:34 config
      32 drwxrwxrwx  3 root      root          16384 Jun 16 02:34 esd
       4 -rwxrwxrwx  1 root      root           1995 Jun 16 02:34 metainfo2.txt
      32 drwxrwxrwx 10 root      root          16384 Jun 16 02:34 mod
      32 drwxrwxrwx  4 root      root          16384 Jun 16 02:34 patches
      16 -rwxrwxrwx  1 root      root           7989 Jun 16 02:34 start

RCC-backup---->
ifs-root-part2 (ASCII): 00f0
3 image type=2(IFS     )  ba0000...1ae05bf (15992256)|93...216 (123):LOCKED
0x00ba0000 - C005F400

Found JTB-07424.01.17118308A2 Backup, will check if all complete...
Partition info...Unable to open device: No such file or directory
Unable to open device: No such file or directory

RCC-backup---->
recover backup from unit......
 -> done :-)

RCC-backup---->
ifs-root-part2 (ASCII): 00f0
3 image type=2(IFS     )  ba0000...1ae05bf (15992256)|93...216 (123):LOCKED
0x00ba0000 - C005F400

Found JTB-07424.01.17118308A2 Backup, will check if all complete...
Partition info... -> Backup is already there!
SWDL............. -> done :-)
Version info..... -> done :-)
Folder lists..... -> done :-)
FEC folders...... -> done :-)
HMI folders...... -> done :-)
Shadow files..... -> done :-)
lsd.jxe.......... -> done :-)
Dataset's........ -> done :-)
EEPROM........... -> done :-)
storage1......... -> done :-)
storage2......... -> done :-)
RCC Flashlock.... -> done :-)
RCC.............. -> done :-)
MMX............../net/mmx/dev/fs0: No such file or directory
 -> failed!
IFS-stage2.......4512+0 records in
4512+0 records out
 -> done :-)
OFFSET's......... -> done :-)
backing up backup -> Backup part 1 is already there! -> Backup part 2 is already there!

ls -als /net/mmx/mnt/boardbook --> total 192
      64 drwxrwxrwx  3 root      root          32768 Jan 01  1970 .
      64 drwxrwxr-x  5 root      root          32768 Jan 01  1970 ..
      64 drwxrwxrwx  3 root      root          32768 Jan 01  1970 MU1329
Free space on /net/mmx/mnt/boardbook 800M


RCC-carp---->
CarPlay and Android Auto activated now

RCC-flash---->
ifs-root-part2 (ASCII): 00f0
3 image type=2(IFS     )  ba0000...1ae05bf (15992256)|93...216 (123):LOCKED
0x00ba0000 - C005F400

Found JTB-07424.01.17118308A2 Backup, will check if all complete...
Partition info... -> Backup is already there!
SWDL............. -> Backup is already there!
Version info..... -> Backup is already there!
Folder lists..... -> Backup is already there!
FEC folders...... -> Backup is already there!
HMI folders...... -> Backup is already there!
Shadow files..... -> Backup is already there!
lsd.jxe.......... -> Backup is already there!
Dataset's........ -> Backup is already there!
EEPROM........... -> Backup is already there!
storage1......... -> Backup is already there!
storage2......... -> Backup is already there!
RCC Flashlock.... -> Backup is already there!
RCC.............. -> Backup is already there!
MMX............../net/mmx/dev/fs0: No such file or directory
 -> failed!
IFS-stage2....... -> Backup is already there!
OFFSET's......... -> Backup is already there!
backing up backup -> Backup part 1 is already there! -> Backup part 2 is already there!

ls -als /net/mmx/mnt/boardbook --> total 192
      64 drwxrwxrwx  3 root      root          32768 Jan 01  1970 .
      64 drwxrwxr-x  5 root      root          32768 Jan 01  1970 ..
      64 drwxrwxrwx  3 root      root          32768 Jan 01  1970 MU1329
Free space on /net/mmx/mnt/boardbook 800M


search File MU1329-ifs-root-part2-0x00ba0000-C005F400.ifs

No Flashfiles there!


RCC-flash---->
ifs-root-part2 (ASCII): 00f0
3 image type=2(IFS     )  ba0000...1ae05bf (15992256)|93...216 (123):LOCKED
0x00ba0000 - C005F400

search JTB-07424.01.17118308A2 Backup File MU1329-ifs-root-part2-0x00ba0000.ifs

[flashlock] unlocking flash: DONEFlashing on OFFSET 0x00ba0000...
Label: PL_qnx_flashit_10495A QNX: RL_qnx_650_10494A Built: 2010/12/10_11-21-47_UTC
/net/rcc/dev/fs0
--------
Flash size: 0x04000000 [= 512 x 0x20000]
Base addr:  0x00ba0000
File size:  0x011a0000 (18481152)
End addr:   0x01d40000
*** erasing:     0x00BA0000 .. 0x01D3FFFF [01D3FFFF] ............................................................................................................................................. 
*** programming: 0x00BA0000 .. 0x01D3FFFF            ............................................................................................................................................. 
*** verifying:   0x00BA0000 .. 0x01D3FFFF            ............................................................................................................................................. 
[flashlock] locking flash
0 image type=1(IPL     )       0...  1ffff (  131072)|0...1 (1):LOCKED
1 image type=2(IFS     )   20000... 3b0e53 ( 3739220)|1...30 (29):LOCKED
2 image type=2(IFS     )  540000... b9db77 ( 6675320)|42...93 (51):LOCKED
3 image type=2(IFS     )  ba0000...1ae05bf (15992256)|93...216 (123):LOCKED
4 image type=4(DSP     ) 3d00000...3dc6923 (  813348)|488...495 (7):LOCKED
5 image type=5(BIOS/ADJ) 3f00000...3ffffff ( 1048576)|504...512 (8):NOT LOCKED
copy EL file from backup JTB-07424.01.17118308A2... -> done :-)

copy FEC file from backup JTB-07424.01.17118308A2... -> done :-)

Reboot Unit...

RCC-carp---->
CarPlay and Android Auto activated now

RCC-flash---->
ifs-root-part2 (ASCII): 00f0
3 image type=2(IFS     )  ba0000...1ae05bf (15992256)|93...216 (123):LOCKED
0x00ba0000 - C005F400

Found JTB-07424.01.17118308A2 Backup, will check if all complete...
Partition info... -> Backup is already there!
SWDL............. -> Backup is already there!
Version info..... -> Backup is already there!
Folder lists..... -> Backup is already there!
FEC folders...... -> Backup is already there!
HMI folders...... -> Backup is already there!
Shadow files..... -> Backup is already there!
lsd.jxe.......... -> Backup is already there!
Dataset's........ -> Backup is already there!
EEPROM........... -> Backup is already there!
storage1......... -> Backup is already there!
storage2......... -> Backup is already there!
RCC Flashlock.... -> Backup is already there!
RCC.............. -> Backup is already there!
MMX............../net/mmx/dev/fs0: No such file or directory
 -> failed!
IFS-stage2....... -> Backup is already there!
OFFSET's......... -> Backup is already there!
backing up backup -> Backup part 1 is already there! -> Backup part 2 is already there!

ls -als /net/mmx/mnt/boardbook --> total 192
      64 drwxrwxrwx  3 root      root          32768 Jan 01  1970 .
      64 drwxrwxr-x  5 root      root          32768 Jan 01  1970 ..
      64 drwxrwxrwx  3 root      root          32768 Jan 01  1970 MU1329
Free space on /net/mmx/mnt/boardbook 800M


search File MU1329-ifs-root-part2-0x00ba0000-C005F400.ifs

No Flashfiles there!

**Reply by luigi1980** [reply]
with any version of mib I proceed the error that generates and always the same Thank you very much

**Reply by harman-f** [reply]
Have you downloaded and copied the patch files to the SD card as well?

Looks like they are missing

**Reply by luigi1980** [reply]
yes I copied everything in the patch folder, just to understand ...... When I patch from the MIB (in the green menu) is normal that the screen darkens for hours? Thank you

**Reply by harman-f** [reply]
No, black screen is not normal.

Also your M.I.B version is kind of old. Get latest one.

MU1329-ifs-root-part2-0x00ba0000-C005F400.ifs is the correct file name. 
So, it does not seem to be on the right place on your SD card.

Please show your folder structure on SD card.

**Reply by luigi1980** [reply]
<img width="471" alt="Immagine 2022-11-03 163840" src="https://user-images.githubusercontent.com/114340127/199766359-8d34935a-4320-458d-9f5d-71fbfff0a2d0.png">
...............here and where I put the patch

**Reply by harman-f** [reply]
That does not look like SD card to me.

https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/wiki/Prepare-M.I.B-SD-card-&-install-M.I.B#add-patch-files

**Reply by luigi1980** [reply]
<img width="960" alt="mib fot 1" src="https://user-images.githubusercontent.com/114340127/199775201-2bb90efc-9147-4930-b366-b52553f895e5.png">
<img width="960" alt="mib fot 2" src="https://user-images.githubusercontent.com/114340127/199775209-7f976149-6138-4e96-9694-b0a2d6899404.png">
In this sd is installed the last mib the path and this in the patch folder I put the folder with the patch right?   thanks for the help

**Reply by harman-f** [reply]
wrong: MU1329-MIB2Q_ER_AUG22_P5092

You train is renamed!

correct: MU1329-M**HI**2Q_ER_AUG22_P5092

You most likely used "M.I.B. -> Advanced Settings -> Block 5F External Access" to rename the train.
Just run it again and patch with correct train.

**Reply by luigi1980** [reply]
sorry I'm impractical ...... So what change the PACHT file should I find MIB2Q?

**Reply by harman-f** [reply]
Best way is to change your train back to normal ...MHI2...
Use this function in GEM to rename it back: M.I.B. -> Advanced Settings -> Block 5F External Access

Afterwards you will be able to patch

**Reply by luigi1980** [reply]
thank you very much I try and let you know if it's okay .......

**Reply by luigi1980** [reply]
perfect everything worked were installed all codes ...... only that car play reads iPhone but does not select ....... It can be a USB mounting error or USB malfunction ??? thanks you're very strong if you wouldn't have helped me I wouldn't have even come this far ...... I attach photos -----
![IMG_8531](https://user-images.githubusercontent.com/114340127/199797871-895f6afc-9f26-4a5c-a2b9-b9f5a4673216.jpg)

**Reply by harman-f** [reply]
check your USB port and make sure that it is not charging only.

Enter REM for FECs
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/wiki/%22hidden-menus%22---key-combinations

**Reply by luigi1980** [reply]
The USB socket and in car play mode and Android Auto virified it on the MIB I tried to reinstall car play but does not connect and everything as in the picture above but now appears the USB also in the media seems as if it was not active .... what do you mean with Enter REM for FECs
thanks again for your patience ...

**Reply by luigi1980** [reply]
Now I do not have the car but I took a picture of what and installed the MIB I'm not sure but I think a code is missing, but I repeat I do not have the car at the moment I check better tomorrow ,,,, it can also be a hardware error???
![IMG_8536](https://user-images.githubusercontent.com/114340127/200044154-fad47c9f-86d6-4340-b589-5a76f93f073c.jpg)

**Reply by luigi1980** [reply]
sorry I wanted to update saying that the fac are installed all in obd11 seems all right ...... Then if there is something else you can try with obd tell me ..... otherwise the only thing that can be is the cable or socket ........... Thank you very much

**Reply by harman-f** [reply]
Most likely hardware related.

**Comment by luigi1980** [comment]
[MU1329-JTB-07424.01.17118308A2.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/9931054/MU1329-JTB-07424.01.17118308A2.log)
this and the log..... I've tried it several times but nothing
